### PR TITLE
migrate to null-safe Dart

### PR DESCRIPTION
**Pull request by yjbanov**

### CODE REVIEW

**Review by hterkelsen — APPROVED** [review]

File: packages/web_drivers/lib/chrome_driver_installer.dart [hunk 1]
```diff
@@ -78,7 +75,7 @@ class ChromeDriverInstaller {
     if (chromeDriverVersion.isEmpty) {
       final int chromeVersion = await _querySystemChromeVersion();
```
Comment (ditman) [inline packages/web_drivers/lib/chrome_driver_installer.dart:76]: It seems that at some point, _querySystemChromeVersion might have returned null? Is this `int?` ?

File: packages/web_drivers/pubspec.yaml [hunk 2]
```diff
@@ -2,11 +2,11 @@ name: web_driver_installer
...
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  args: ^1.5.2
-  http: ^0.12.0
-  path: ^1.6.4
-  test: ^1.6.5
-  yaml: ^2.2.0
+  args: ">=1.5.2 <3.0.0"
+  http: ">=0.12.0 <1.0.0"
```
Comment (ditman) [inline packages/web_drivers/pubspec.yaml:9]: This is not preventing you from picking up 0.13, 0.14... which are major versions according to pub. I think the upper limit here should be `<0.14.0`?

File: packages/web_drivers/lib/chrome_driver_installer.dart (hunk 1 above)
Comment (yjbanov) [reply packages/web_drivers/lib/chrome_driver_installer.dart:76]: It seems to know if it's now able to find a version.

File: packages/web_drivers/pubspec.yaml (hunk 2 above)
Comment (yjbanov) [reply packages/web_drivers/pubspec.yaml:9]: We use a git dependency on this package from the engine, which uses tight constraints, so it won't let these version constraints go too far. Honestly, I'm tempted to make everything `any` here, just for simplicity.

**Pull request merged**